.github/workflows/branch-name-policy.yml

```diff
@@ -0,0 +1,16 @@
+name: Fail when frozen files are changed in a PR
+on:
+  pull_request:
+    branches:
+    - '*/**'
+    - '**/*
```

Comment:
So, this is interesting... In my understanding, `*` is all characters _except_ slashes, whereas `**` also matches slashes.

Would be curious if `**/**` would work - i.e. whether `**` is matching greedily or not, but it seems fine/safer the way it is now :D